virttest/virt_vm.py

```diff
@@ -937,6 +937,13 @@ def add_nic(self, **params):
         """
         if 'nic_name' not in params:
             params['nic_name'] = utils_misc.generate_random_id()
+
+        # 'romfile' is not a nic device cartesian param, remove it because it
+        # can overwrite the romfile attribute of a nic device, use 'nic_param'
+        # to set romfile attribute
+        if 'romfile' in params:
+            del params['romfile']
+
```

Comment:
we don't need this code change, maybe.
if we don't set romfile in cfg, it should work fine.
what do you think @luckyh

Reply:
@zhencliu yes, agree. Let's remove it.

Reply:
removed, thanks